alacritty.yml

```
@@ -312,6 +319,13 @@
   #
   #indexed_colors: []
 
+  # Transparent backgrounds with transparency enabled
+  #
+  # Whether or not `window.opacity` applies to all backgrounds or only to the
+  # default background. When set to `false` all cells will be transparent
+  # regardless of their background color.
```

```suggestion
  # Whether or not `window.opacity` applies to all cell backgrounds or only to the
  # default background. When set to `false` all cells will be transparent
  # regardless of their background color.
```

I think explicitly pointing out cells here makes sense, since it is possible to change the terminal background in general. We don't just mean that a changed terminal background is transparent, but that each individual cell background is.

We could also change "default background" to "terminal background". I think many people probably don't see a "normal" cell as a cell with a default background but rather a cell with a transparent background.

The second part of the paragraph is also completely wrong. When set to `false`, cells that do not have the default background are **opaque**, not transparent.

```suggestion
  # Whether or not `window.opacity` applies to all backgrounds or only to the
  # default background. When set to `true` all cells will be transparent
  # regardless of their background color.
```

---

This is also still unchanged.